### PR TITLE
Feature: Multi-object detection in VRS, run VRS processing w/out QR code; enhanced: logging, cleaned up print commands

### PR DESCRIPTION
**Pull request by zephirefaith**
Check visuals in Slack channel.

### CODE REVIEW

File: aria_data_loaders/aria_data_utils/detector_wrappers/object_detector.py [hunk 1]
```diff
@@ -161,9 +175,25 @@ def process_frame(self, img_frame: np.ndarray):
         ) = self._get_scored_object_detections(
             img_frame, heuristic=self._aria_fetch_demo_heuristics
         )
-        if stop and stop["penguin_plush"]:
-            print("Turning off object-detection")
-            self.disable_detector()
+        if stop != {}:
```
Comment (rutadesai) [inline aria_data_loaders/aria_data_utils/detector_wrappers/object_detector.py:178]: Didnt understand this, why and when do we stop detection? May be add a comment?

**Review by rutadesai — APPROVED** [review]
Looks good to me! Thanks for testing this out :)

File: aria_data_loaders/aria_data_utils/aria_streamer.py [hunk 2]
```diff
@@ -795,102 +795,154 @@ def get_avg_spotWorld_T_marker(
...
+    logger = logging.getLogger("VRS_MPS_STREAMER")
     vrsfile = os.path.join(data_path, vrs_name + ".vrs")
     vrs_mps_streamer = AriaReader(
         vrs_file_path=vrsfile, mps_file_path=data_path, verbose=verbose
     )
 
     outputs = vrs_mps_streamer.parse_camera_stream(
         stream_name=STREAM1_NAME,
-        detect_qr=True,
+        detect_qr=qr,
```
Comment (KavitShah1998) [inline aria_data_loaders/aria_data_utils/aria_streamer.py:819]: amazing!

**Review by KavitShah1998 — APPROVED** [review]

**Comment by KavitShah1998** [comment]
LGTM, we would need to address `FIXME` soon or create issues for them. @zephirefaith

**Pull request merged**